lib/dialects/abstract/query-generator.js

```diff
@@ -21,7 +20,7 @@ const QueryGenerator = {
     tableName = tableName || {};
     return {
       schema: tableName.schema || options.schema || 'public',
-      tableName: _.isPlainObject(tableName) ? tableName.tableName : tableName,
+      tableName: Utils._.isPlainObject(tableName) ? tableName.tableName : tableName,
```

Comment:
Okay, so can we move completely away from using `Utils._.lodashMethod` and  change everything to be `_.lodashMethod`?